testing/integration/monitoring_logs_test.go

```diff
@@ -15,49 +15,34 @@ import (
 	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
-	"github.com/stretchr/testify/suite"
 
 	"github.com/elastic/elastic-agent-libs/kibana"
+	"github.com/elastic/elastic-agent/pkg/control/v2/client"
 	atesting "github.com/elastic/elastic-agent/pkg/testing"
 	"github.com/elastic/elastic-agent/pkg/testing/define"
 	"github.com/elastic/elastic-agent/pkg/testing/tools"
+	"github.com/elastic/elastic-agent/pkg/testing/tools/check"
 )
 
-func TestEnrollAndLog(t *testing.T) {
+func TestMonitoringLogsShipped(t *testing.T) {
 	info := define.Require(t, define.Requirements{
 		OS: []define.OS{
-			{Type: define.Linux},
+			{Type: define.Linux,
```

Comment:
FYI the framework now supports doing this from the command line https://github.com/elastic/elastic-agent/blob/main/docs/test-framework-dev-guide.md#selecting-specific-platform